kolibri/core/assets/src/views/NotificationsRoot.vue

```diff
@@ -95,14 +95,11 @@
     },
     data() {
       return {
+        notifications: [],
         notificationModalShown: true,
       };
     },
     computed: {
-      ...mapState({
-        error: state => state.core.error,
```

Comment:
I'm not seeing why `error` was removed here, was this intentional?

Reply:
mapState was removed to completely decouple the component from vuex store, but i did not think of error handling. Should i handle the error locally in `getnotification` method or keep the `mapState` for error property?

Reply:
@nucleogenesis i removed the empty bashrc files. As for the error handling, it is being handled locally in the methods. If you want me to handle error with `mapState`, should i do that?

Reply:
My apologies for missing this last week @iamshobhraj 

The `mapState` function is assigning `this.error` to the value of `state.core.error` so that `this.error` is reactive whenever `state.core.error` changes.

Elsewhere in the component, `this.error` [is expected to exist](https://github.com/learningequality/kolibri/blob/90113523576c04e585d54d41e8c8224e20dc3c14/kolibri/core/assets/src/views/NotificationsRoot.vue#L106-L109) so we should keep `mapState` imported and keep this particular line mapping to the `error`.

To be a bit more clear, basically you just need to remove the line `notifications: ...` but not the `error: ...` line.